test/vault/secrets/kvv2_test.clj

```diff
@@ -218,7 +219,7 @@
                (is (= :post (:method req)))
                (is (= (str vault-url "/v1/" mount "/delete/" path-passed-in) (:url req)))
                (is (= token-passed-in (get (:headers req) "X-Vault-Token")))
-               (is (= {:versions [12 14 147]} (:form-params req)))
+               (is (= (json/generate-string {:versions [12 14 147]}) (:body req)))
```

Comment:
I usually find serialization of the expected value like this to kind of be a test smell - you should be testing the actual string that is generated, _or_ if ordering determinism is a problem, you should parse the body and assert it equals a data structure. Otherwise you're kind of including some of the logic you're supposed to be testing in the test itself.